internal/service/flexcluster/resource_test.go

```diff
@@ -115,16 +116,23 @@ func configBasic(projectID, clusterName, provider, region string, terminationPro
 		data "mongodbatlas_flex_cluster" "test" {
 			project_id = mongodbatlas_flex_cluster.test.project_id
 			name       = mongodbatlas_flex_cluster.test.name
+		}
+		data "mongodbatlas_flex_clusters" "test" {
+			project_id = mongodbatlas_flex_cluster.test.project_id
 		}`, projectID, clusterName, provider, region, terminationProtectionEnabled)
 }
 
-func checksFlexCluster(projectID, clusterName string, terminationProtectionEnabled bool) resource.TestCheckFunc {
+func checksFlexCluster(projectID, clusterName string, terminationProtectionEnabled bool, flexClusterCount int) resource.TestCheckFunc {
 	checks := []resource.TestCheckFunc{checkExists()}
 	attrMap := map[string]string{
 		"project_id":                     projectID,
 		"name":                           clusterName,
 		"termination_protection_enabled": fmt.Sprintf("%v", terminationProtectionEnabled),
 	}
+	pluralMap := map[string]string{
+		"project_id": projectID,
+		"results.#":  "1",
```

Comment:
is `flexClusterCount` param needed? I believe we are assuming config will only have 1 flex cluster

Reply:
Agreed, removed `flexClusterClount`